src/interpreter/Interpreter.cpp

```diff
@@ -488,11 +445,10 @@ static void initAddressToOpcodeTable()
 ByteCodeStackOffset* Interpreter::interpret(ExecutionState& state,
                                             size_t programCounter,
                                             uint8_t* bp,
-                                            Instance* instance,
-                                            Memory** memories,
-                                            Table** tables,
-                                            Global** globals)
+                                            Instance* instance)
 {
+    Memory** memories = reinterpret_cast<Memory**>(reinterpret_cast<uintptr_t>(instance) + Instance::alignedSize());
```

Comment:
Why don't you initialize `memories` as `instance->m_memories`?

Reply:
And is it enough to initialize local `memories` only?
Other structures like tables, globals don't need local variables since they are not accessed frequently like memories, right?

Reply:
Why don't you initialize memories as instance->m_memories?
-> I tested two way of init memories.
. access instance->m_memories -> needs memory access
. computing address(reinterpret_cast<Memory**>(reinterpret_cast<uintptr_t>(instance) + Instance::alignedSize())) -> needs ALU on CPU

the second way shows better performance when I tested

And is it enough to initialize local memories only?
-> Yes. only memories is used frequently when I tested
but globals and tables are don't. but it needs a space on stack